binary/grakn-core.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Grakn Core
+[Service]
+Type=simple
+ExecStart=/opt/grakn/core/grakn server
+Restart=on-failure
```

Comment:
Let's not do restart since it would be too magical for our users @vmax

Reply:
@lolski fixed in https://github.com/graknlabs/common/pull/102/commits/55f013e75fb717bdf9e2e68ba3c8c277207ec77a